src/main/java/controller/KeyUtils.java

```diff
@@ -8,23 +8,34 @@
 import java.io.IOException;
 import java.io.UnsupportedEncodingException;
 import java.math.BigInteger;
+import java.net.URISyntaxException;
 
-import com.google.bitcoin.core.Base58;
-import com.google.bitcoin.core.ECKey;
-import com.google.bitcoin.core.ECKey.ECDSASignature;
-import com.google.bitcoin.core.Sha256Hash;
-import com.google.bitcoin.core.Utils;
+import org.bitcoinj.core.Base58;
+import org.bitcoinj.core.ECKey;
+import org.bitcoinj.core.ECKey.ECDSASignature;
+import org.bitcoinj.core.Sha256Hash;
+import org.bitcoinj.core.Utils;
 
 public class KeyUtils {
 
     final private static char[] hexArray = "0123456789abcdef".toCharArray();
-    final private static String PRIV_KEY_FILENAME = "bitpay_private.key";
-    
+    final private static String PRIV_KEY_FILENAME = "/bitpay_private.key";
+
     public KeyUtils() {}
-    
+
     public static boolean privateKeyExists()
     {
-        return new File(PRIV_KEY_FILENAME).exists();
+        return getPrivateKeyFile().exists();
```

Comment:
yeah, needed this in production - I'm running in jetty so reading from classpath is better than absolute paths which differ between dev/test/prod. Agreed on the NPE, let me know if you want me to fix it, not sure what you did in the other pull request.
Note: my maven pull request of the version 1 bitpay client was never merged so I did not think anything would happen with this one, thanks for picking it up ;)